src/layer/__tests__/S2L1CLayer.ts

```diff
@@ -1,5 +1,200 @@
-import { S2L1CLayer, ApiType } from '../../index';
 import '../../../jest-setup';
+import axios from 'axios';
```

Comment:
With support for catalog service on other datasets, the tests for them should also be written. However I am afraid of code duplication here - maybe we can find some smart way of using only different fixtures (responses, params,...) to test each of the datasets?

Also, IIUC, there are some issues still open on the backend which mean that this patch can't be merged yet. Maybe writing the tests would help us identify when the issues are resolved and we can merge this?